### DISCUSSION TITLE
Error: “” is not a valid LUIS application id. Bot deployed via ADO into Azure subscription

### DISCUSSION THREAD

**Discussion by puneetsohi** [microsoft/BotFramework-Composer#5769] [General]
I'm building a bot through Bot Composer (using Luis for LU). 
Exported the bot into a VS Project
Deployed it using Azure DevOps pipeilne into Azure Subscription
Tested it using "Test in Web Chat" functionality
Got the error: “” is not a valid LUIS application id.

**Comment by a-b-r-o-w-n** [comment]
@puneetsohi We are tracking this as an issue in #5448. Closing this -- please follow along in that issue.
